package.json

```diff
@@ -35,12 +35,12 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "cliui": "^3.2.0",
+    "cliui": "^8.0.1",
```

Comment:
why can't we stick with v7 and avoid the breaking change?

Reply:
Yep, I'm not a fan of breaking changes either. Let me make this change; Thanks for the feedback

Reply:
I've updated the PR and the PR description with the newest changes. Thanks, much simpler